ProcessMaker/Repositories/TokenRepository.php

```diff
@@ -159,6 +161,24 @@ public function persistActivityActivated(ActivityInterface $activity, TokenInter
         $this->instanceRepository->persistInstanceUpdated($token->getInstance());
     }
 
+    /**
+     * Get action by email configuration
+     *
+     * @param ActivityInterface $activity
+     */
+    private function getAcionByEmail(ActivityInterface $activity)
```

Comment:
Could you add a test for this method please?

Reply:
We will have another ticket for the test of this function, when saving all the data in the xml of a process is completed or fixed.